packages/angular_devkit/schematics/src/tree/host-tree.ts

```diff
@@ -16,6 +17,7 @@ import {
   normalize,
   virtualFs,
 } from '@angular-devkit/core';
+import { parse } from 'jsonc-parser';
```

Comment:
Nit: Consider aliasing this to `parseJsonc` or something similar, as `parse` is very generic and not clear what it would represent when called from the tree.